themes/openy_themes/openy_carnation/src/scss/presentation/_variables.scss

```diff
@@ -30,6 +30,7 @@ $yellow:              #fcaf17;
 $green:               #20bdbe;
 $teal:                #006b6b;
 $cyan:                #01a490;
+$header-grey:         #2f2f2f;
```

Comment:
Move to the other section. This is not an appropriate place. "header grey" is not a "open y color palette" color.

Also, use `$gray-700`:
`$header-grey: $gray-700`

Also, it's not `header grey`, it's `default header color`

Reply:
@AndreyMaximov Fixed